dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,18 @@ def load_user
     @user = current_user
   end
 
+  # GET /api/v1/users/<user_id>/school_name
+  def get_school_name
+    render json: {
+      school_name: current_user&.school
```

Comment:
this returns the entire school object, so i think we want to use `current_user&.school&.name` here if we just want to return the school name